zip/preamble_test.go

```diff
@@ -30,6 +30,14 @@ func TestPreamble(t *testing.T) {
 			ZipFile:          "zip_preamble.zip",
 			PreambleChecksum: "038a57f3f807fa91bdd30239b9711fccf0d782fe2f036e03211852237e94d24c", // another.zip
 		},
+		{
+			ZipFile:          "arcat_preamble.zip",
+			PreambleChecksum: "46533b2dfa35ad537d3561ebee0c7af8941bc65363c1b188e1be6eaf79e9138c", // shebang_twolines.txt
+		},
```

Comment:
This zip file was written by arcat (i.e. by `archive/zip`), whereas the other test zip files here were simply `cat`ted onto another file acting as the preamble. This meant that the byte offsets of the CDFH and EOCD entries in those zip files were wrong, so the presence of the preamble was easily detectable by the previous version of `Preamble`. The main use case for the `--preamble_from` option is in fact to write the shebang from one pex file into another pex file in the python-rules plugin, so it's important that preambles can be read correctly from zip files written by arcat.